kotlin/build.gradle

```diff
@@ -1,7 +1,9 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 apply from: '../version.gradle'
 
-buildscript {
+buildscript { 
+    ext.kotlin_version = '1.3.50'
```

Comment:
Same with the other sample, this slipped in automatically from android studio and needs to be fixed